ManageIQ/manageiq
# [WIP] Add Introspection support for OpenStack infrastructure providers

This patch adds menu options allowing admins to introspect hosts.
Only hosts in "manageable" state can be introspected.

Requires
* https://github.com/ManageIQ/manageiq/pull/10644
* https://github.com/ManageIQ/manageiq/pull/10860

## Steps for Testing/QA [Optional]

1. Register a new node. The node should be in manageable state and it's introspected attribute is set to false.
2. Go to host/node and select Introspect.
3. A message indicating introspect has been queued will be displayed.
4. After introspection has completed, a refresh is initiated.
5. After refresh is complete, you should see that the introspected attribute is now true.
